Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:wheezy
+MAINTAINER Charles Flèche <charles.fleche@free.fr>
+
+RUN apt-get update \
+ && apt-get -y install python-pip \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --upgrade pip
+
+COPY requirements/requirements.txt /tmp/requirements.txt
+
+RUN apt-get update \
+ && apt-get -y install \
+      mysql-client \
+      postgresql-client \
+      sqlite3 \
+      libxml2-dev \
+      libxslt-dev \
+      libpq-dev \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+ && apt-get -y install \
+      python-dev \
+      --no-install-recommends
+
+RUN pip install -r /tmp/requirements.txt
+
+WORKDIR /usr/src/commcare-hq
+
+COPY . /usr/src/commcare-hq
```

Comment:
Totally. Right now I'm focusing on deploying a ready to use image for trainings / offline installs, but next time I modify CommCare codes I'll take a look at mounting a clone in a container.